webpack/components/sections/SectionQuestionList.js

```diff
@@ -16,21 +19,27 @@ class SectionQuestionList extends Component {
         <div>Loading...</div>
       );
     }
+    let currentUserId = this.props.currentUser ? this.props.currentUser.id : -1;
     return (
       <div className="question-group">
         {this.props.questions.map((q, i) => {
-          if (q.status === 'published' || q.createdById === this.props.currentUserId) {
-            return <SearchResult key={i} type='question' result={{Source: q}} programVar={q.programVar} currentUser={{id: -1}} />;
+          if (q.status === 'published' || q.createdById === currentUserId) {
+            return <SearchResult key={i} type='question' result={{Source: q}} programVar={q.programVar} currentUser={this.props.currentUser} updatePDV={this.props.updatePDV}/>;
           }
         })}
       </div>
     );
   }
 }
 
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators({updatePDV}, dispatch);
+}
+
 SectionQuestionList.propTypes = {
   questions: PropTypes.arrayOf(questionProps),
-  currentUserId: PropTypes.number
+  updatePDV: PropTypes.func,
+  currentUser: PropTypes.object
 };
 
-export default SectionQuestionList;
+export default connect(null, mapDispatchToProps)(SectionQuestionList);
```

Comment:
This means the file should be moved into the `containers/sections` folder.

Reply:
good catch. changed as requested.